pipcompilemulti/features/controller.py

```diff
@@ -61,9 +64,11 @@ def save_command_options(*args, **kwargs):
 
     def pin_options(self, env_name):
         """Return list of options to pin command."""
-        options = self.use_cache.pin_options()
+        options = []
```

Comment:
@peterdemin a copy is necessary here otherwise `options` is a reference and we're extending (and thus polluting) the class `enabled_pin_options` or `disabled_pin_options` lists.

I felt initializing `options` to an empty list and then extending was cleaner than, 

```
options = self.use_cache.pin_options().copy()
```